astropy/stats/tests/test_funcs.py

```diff
@@ -256,3 +258,94 @@ def test_gaussian_sigma_to_fwhm():
 def test_gaussian_sigma_to_fwhm_to_sigma():
     assert_allclose(funcs.gaussian_fwhm_to_sigma *
                     funcs.gaussian_sigma_to_fwhm, 1.0)
+
```

Comment:
It would be good to add at least one (possible more) tests here where the *exact* (up to numerical errors, so say to within 1e-7) answer is known, not just test `>0.05` or similar. If the results were just a little wrong, we would not notice with the current tests.
That would of course require to hardcode an input table instead of using random numbers. Maybe there is an example in one the papers you cited?